presto-product-tests/conf/tempto/tempto-configuration-for-docker-kerberos-simba.yaml

```diff
@@ -0,0 +1,47 @@
+hdfs:
```

Comment:
I think should have another config file for running simba driver tests without kerberos